plugins/auth/index.js

```diff
@@ -108,7 +110,7 @@ exports.register = (server, options, next) => {
 
             server.auth.strategy('session', 'cookie', {
                 cookie: 'sid',
-                ttl: 12 * 60 * 60 * 1000, // 12 hours in milliseconds
+                ttl: 2 * 60 * 60 * 1000, // 2 hours in milliseconds
```

Comment:
Don't know if we should change this. If we change this, then users need to log in for every 2 hours.. it's kinda annoying

Reply:
I don't think this ttl is for the user session, so it should be okay

Reply:
Asked @petey about this. This is for transaction between API and github to make sure the calls come from the same user.